js/feed.js

```diff
@@ -1,141 +1,198 @@
-(function () {
-  const intervals = [
-    { label: "year", seconds: 31536000 },
-    { label: "month", seconds: 2592000 },
-    { label: "day", seconds: 86400 },
-    { label: "hour", seconds: 3600 },
-    { label: "minute", seconds: 60 },
-    { label: "second", seconds: 0 }
-  ];
+function createActivityFeed(amountOfPosts) {
+  $(document).ready(function () {
+    const intervals = [
+      { label: "year", seconds: 31536000 },
+      { label: "month", seconds: 2592000 },
+      { label: "day", seconds: 86400 },
+      { label: "hour", seconds: 3600 },
+      { label: "minute", seconds: 60 },
+      { label: "second", seconds: 0 }
+    ];
 
-  function timeSince(date) {
-    const seconds = Math.floor((Date.now() - date.getTime()) / 1000);
-    const interval = intervals.find(i => i.seconds < seconds);
-    const count = Math.floor(seconds / interval.seconds);
-    return count + " " + interval.label + (count !== 1 ? "s" : "") + " ago";
-  }
+    function timeSince(date) {
+      const seconds = Math.floor((Date.now() - date.getTime()) / 1000);
```

Comment:
I didn't see this before, please don't use ES6 syntax because we don't transpile the code and we need to be backward compatible, use `var`